kalite/distributed/management/commands/setup.py

```diff
@@ -398,7 +395,7 @@ def handle(self, *args, **options):
         ########################
 
         # Move database file (if exists)
-        if install_clean and database_file and os.path.exists(database_file):
+        if install_clean and database_file and database_exists:
```

Comment:
This is the essential part, but I believe that the expression `install_clean and database_file and database_exists` is pretty messed up when looking at the definitions of these vars above.